### PR TITLE
Bump version to 0.4.3

### PR DESCRIPTION
**Pull request by chrisduerr**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Crash when writing a fullwidth character in the last column with auto-wrap mode disabled
 - Crashing at startup on Windows
 
-## 0.4.2-dev
+## 0.4.2
```
Comment (chrisduerr) [inline CHANGELOG.md:22]: Looks like this got messed up during rebase.

**Comment by chrisduerr** [comment]
Thanks for reminding me!

**Comment by chrisduerr** [comment]
@mmstick just letting you know 0.4.3 has been released. Is the PPA still maintained? It wasn't bumped to 0.4.2, but I'd assume you were just busy?

**Comment by mmstick** [comment]
Busy with 20.04 + Pop Shell, but I'll do this today

**Comment by chrisduerr** [comment]
Thanks!

**Pull request merged**